pkg/reconciler/v1alpha1/namespace/namespace.go

```diff
@@ -240,7 +241,7 @@ func injectedLabels() map[string]string {
 	}
 }
 
-func (r *reconciler) reconcileBrokerFilterRBAC(ctx context.Context, ns *corev1.Namespace, sa *corev1.ServiceAccount) (*rbacv1.RoleBinding, error) {
+func (r *reconciler) reconcileBrokerFilterRBAC(ctx context.Context, ns *corev1.Namespace, sa *corev1.ServiceAccount) (*rbacv1.ClusterRoleBinding, error) {
```

Comment:
The main change to this file is setting ClusterRoleBinding in order to make the e2e test to pass.
Should be removed afterwards with Adam's fix...
The rest of the changes is just formatting